src/Microsoft.AspNet.Identity/SignInManager.cs

```diff
@@ -41,7 +42,7 @@ public class SignInManager<TUser> where TUser : class
             {
                 throw new ArgumentNullException(nameof(userManager));
             }
-            if (contextAccessor == null || contextAccessor.HttpContext == null)
```

Comment:
Maybe we can null check in the getter then so the user gets a clearer error message if the context it is null when it's actually needed. I guess whether or not it's worth it depends on how possible it is for this issue to come up.